packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -82,16 +82,45 @@ RowRenderer.propTypes = {
 	titleProps: ItemTitle.propTypes.titleProps,
 };
 
+function headerContent(column, sort) {
+	if (!sort) {
+		return column.label;
+	}
+
+	const isCurrentSortField = sort.field === column.key;
+	const icon = isCurrentSortField ? 'talend-caret-down' : null;
+	const iconTransform = sort.isDescending ? 'rotate-180' : null;
+	const onChange = event => sort.onChange(
+		event,
+		{
+			field: column.key,
+			isDescending: isCurrentSortField ? !sort.isDescending : false,
```

Comment:
I guess you could do most of the operations on the constants into a single outside function.